### PR TITLE
Add tarka to ignore list.

### PR DESCRIPTION
**Pull request by tarka**
##### SUMMARY

Add `tarka` to bot ignore list.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
jira.py

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -679,7 +679,7 @@ files:
   $modules/jenkins_script.py:
     maintainers: hogarthj
   $modules/jira.py:
-    ignore: DWSR
+    ignore: DWSR tarka
     labels: jira
     maintainers: Slezhuk tarka pertoft
```
Comment (felixfontein) [inline .github/BOTMETA.yml:684]: You also need to remove yourself here:
```suggestion
    maintainers: Slezhuk pertoft
```

**Review by felixfontein — COMMENTED** [review]
Thanks a lot for all your contributions and for maintaining this module!

File: .github/BOTMETA.yml (hunk 1 above)
Comment (tarka) [reply .github/BOTMETA.yml]: Thanks @felixfontein. PR updated.

**Comment by felixfontein** [comment]
@tarka thank you very much!

**Pull request merged**